contents/blog/the-posthog-array-1-31-0.md

```diff
@@ -0,0 +1,125 @@
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-correlation.png" alt="TODO: Change this image" />
+
+
+<br />
+
+### Improved user experience
+
+We've been working hard on creating the best user experience, particularly in our core Analytics product. Creating and navigating insights will now be a lot faster and smoother. Improvements include: finding events & properties faster, overall faster navigation, improvements to tooltips, etc.
+
+<br />
+
+
+@yakkomajuri Special Migrations context here?
+
+
+### Other improvements & fixes
+- Improved breakdown in funnels. We have a new vertical layout to display funnels broken down by a property (e.g. conversion by browser) that will make it even easier to compare conversion across multiple dimensions.
+- Searching events & properties is now significantly faster. We've changed the way these properties are shown and we now list them by popularity within your team. Create those insights faster!
+- Following our [lemonade redesign](/blog/the-posthog-array-1-30-0#fresh-new-look-and-feel) we now have a ton more UI improvements. In particular tables got a pretty slick facelift.
+- We've improved the layout configuration for dashboards so they'll be more responsive and look better on very small or very large screens.
+- Pushed significant improvements to the recordings list load time.
+- Introducing a new funnel query builder. We now group advanced options separately, so you funnel definition is clearer.
+- Fixed bug in feature flags when in certain cases a 0% release was considered as a 100% release.
+- Fixed bug where private project names were shown to members who shouldn't have access.
+- Plus 350+ improvements & fixes.
+
+### Deprecation & removal notices
+
+1. This version (`1.31.0`) no longer supports a Postgres-only deployment of PostHog. Read [our migration guide](/docs/self-host/migrate-from-postgres-to-clickhouse) for instructions on moving over to a ClickHouse version. ClickHouse provides faster queries and is optimized for very large volumes of data, and you will also get a new lot of features.
+2. We're deprecating the **Sessions** insight (distribution of session length). Please [reach out](/support) if you have any feedback on this.
+
+### Talk to us on how we can improve
+
+We’re always working on improving the product experience and would love to talk to you! Please join one of our Product, Engineering, or Marketing team members on a quick 30-min call to help us understand how to improve. Schedule directly [on Calendly](https://calendly.com/posthog-feedback).
+
+As a small thank you for your time, we're giving away awesome [PostHog merch](https://merch.posthog.com)!
+
+
+## PostHog News
+
+Welcome Cameron DeLeone! Cameron joined PostHog to help us level up our Customer Success experience. Cameron is ?? (🍍 on 🍕).
```

Comment:
@camerondeleone what's your stance on pineapple on pizza and a fun fact about you that you'd like to share?

Reply:
@paolodamico Pineapple on pizza: definitely not. I guess it's a valid culinary move, it's just not... pizza.

Fun fact: I've always been a food lover, and started talking about food at 7 months old (my first word was "broc" for broccoli). I haven't shut up about it since.

Reply:
Love it, thanks!